### DISCUSSION TITLE
Funding tag can now be at the item level

### DISCUSSION THREAD

**Discussion by daveajones** [Podcastindex-org/podcast-namespace#664] [Announcements - Changes Made]
The `<podcast:funding>` tag is now valid a the `<channel>` or `<item>` level.

**Comment by nathangathright** [comment]
Should we add some language around the expected use? (à la the person tag?)

> It is suggested that <channel> is always populated, and <item> is populated where needed for an individual episode. Where present, people information in <item> wholly replaces all information from the <channel>.
>
> Publishers are expected to use the podcast:person element in the <channel> parent to set the regular people involved in the podcast: the detail that would be expected to be seen in an overview of the show.
>
> Publishers are expected to use the podcast:person in the <item> parent to replace all existing information for an individual episode.

**Comment by samsethi** [comment]
Hi 

TrueFans has already added support for the item-level funding tag. At the moment this points to an external payment provider but we are going to enable podcasters to setup an internal funding amount in either Fiat Currency or SATs.  

<img width="1295" alt="TrueFans___Shows___Podnews_Weekly_Review_🔊" src="https://github.com/user-attachments/assets/4ec2d356-d54b-44ab-8361-41e38c54ab58">
